README.md

```diff
@@ -28,11 +28,46 @@ $ git clone https://github.com/prebid/prebid-cache-java.git
 ```
 
 (2). Start Redis or Aerospike:
+If you have installed [Redis](https://redis.io/docs/install/install-redis/) or [Aerospike](https://aerospike.com/docs/server/operations/install) locally, you may start them both (or separately, depends on your needs) via bash:
+```bash
+$ sudo systemctl start redis.service & sudo systemctl start aerospike
```

Comment:
1) Please remove the `$` symbol from all command examples as this will break copy-pasting.
2) Better to list both commands on a separate lines so people won't think it's the part of one command.
``` bash
sudo systemctl start redis
sudo systemctl start aerospike
```